dev/tests/integration/testsuite/Magento/Vault/_files/payment_active_tokens.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+include "customer.php";
```

Comment:
Please implement `rollback` script to remove test data persisted by this fixture.